nowcasting_dataset/config/model.py

```diff
@@ -372,8 +429,10 @@ class Process(BaseModel):
 
     @validator("local_temp_path")
     def local_temp_path_to_path_object_expanduser(cls, v):
-        """Convert the path in string format to a `pathlib.PosixPath` object
-        and call `expanduser` on the latter."""
+        """Convert the path in string format to a `pathlib.PosixPath` object.
```

Comment:
this change was needed to get the linter to pass

Reply:
Yea sorry about that - that must have sneaked through